bfvmm/src/vmcs/src/vmcs_intel_x64_check_guest.cpp

```diff
@@ -1492,7 +1492,7 @@ vmcs_intel_x64::check_guest_rflags_reserved_bits()
 void
 vmcs_intel_x64::check_guest_rflags_vm_bit()
 {
-    if (!is_enabled_ia_32e_mode_guest() && vmcs::guest_cr0::protection_enable::get() == 1)
+    if (!vmcs::vm_entry_controls::ia_32e_mode_guest::is_enabled() && vmcs::guest_cr0::protection_enable::get() == 1)
```

Comment:
Look for instances of this where the one line if statement is compound and just turn them into two separate checks for two reasons:
- This ensures coveralls is checking everything (one liners can hide one of the checks)
- it's too verbose